xcdat/regridder/accessor.py

```diff
@@ -76,22 +76,15 @@ def grid(self) -> xr.Dataset:
 
         return ds
 
-    def _get_axis_data(
-        self, name: str, standard_name: str
-    ) -> Tuple[xr.DataArray, xr.DataArray]:
-        try:
-            axis = self._ds.cf[name]
-        except KeyError:
-            raise KeyError(
-                f"{standard_name} axis could not be correctly identified in the Dataset"
-            )
```

Comment:
This logic is already implemented with `get_axis_coord()`, so I chose to reuse it here.